### PR TITLE
[rendergraph] Updated to version 1.2.0.

### LINKED ISSUE #29358: [rendergraph] update to 1.2.0
Library name: RenderGraph

New version number: 1.2.0

### PR DESCRIPTION
**Pull request by DragonJoker**
Fixes #29358

- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [x] The "supports" clause reflects platforms that may be fixed by this new version
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/rendergraph/vcpkg.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "rendergraph",
-  "version": "1.1.0",
+  "version": "1.2.0",
   "description": "Vulkan render graph implementation.",
   "homepage": "https://github.com/DragonJoker/RenderGraph",
   "license": "MIT",
-  "supports": "!x86 & !uwp",
+  "supports": "!uwp",
```
Comment (JonLiu1993) [inline ports/rendergraph/vcpkg.json:7]: @DragonJoker, Thanks for your PR, could you please provide evidence that the new version supports X86?

Comment (DragonJoker) [reply ports/rendergraph/vcpkg.json:7]: Well, at first I've worked on this library in x64 only, hence it didn't compile in x86.
I'vestarted working on an Android port of my 3D engine, which relies on Ashes.
Hence I've had to port this library too, for amr8 (64bits) and arm7 (32bits).
That's the only evidence I have.

Comment (JonLiu1993) [reply ports/rendergraph/vcpkg.json:7]: Thanks for the clarification.

**Review by JonLiu1993 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**